docs/src/test-components-js.md

```diff
@@ -234,7 +234,7 @@ This includes any config that needs to be run before or after mounting the compo
 
   ```js title="src/pages/ProductsPage.spec.tsx"
   import { test, expect } from '@playwright/experimental-ct-react';
-  import type { HooksConfig } from '@playwright/test';
+  import type { HooksConfig } from 'playwright';
```

Comment:
Most of our customers don't have `playwright` installed and also `playwright` has nothing to do with the test runner, so should not be aware of hooks configs. What are we fixing here?

Reply:
This is referencing to `/{root-of-the-project/playwright` eg: https://github.com/microsoft/playwright/blob/main/tests/components/ct-react-vite/playwright/index.tsx#L5

But i see now how this is confusing, will adjust it